packages/flutter/lib/src/animation/curves.dart

```diff
@@ -1385,6 +1385,24 @@ class Curves {
   /// {@animation 464 192 https://flutter.github.io/assets-for-api-docs/assets/animation/curve_fast_linear_to_slow_ease_in.mp4}
   static const Cubic fastLinearToSlowEaseIn = Cubic(0.18, 1.0, 0.04, 1.0);
 
+  /// A curve that starts slowly, speeds up very quickly, and then ends slowly.
+  ///
+  /// This curve is used by default to animate page transitions used by CupertinoPageRoute.
```

Comment:
Link it: `[CupertinoPageRoute]`.

Also, maybe it should be on the next line (but it's hard to tell on GitHub).